plans/all.fmf

```diff
@@ -19,6 +28,9 @@ prepare:
   - name: Show test system information
     how: shell
     script: free -m; lsblk -i; lscpu; cat /proc/1/sched
+  - name: Show TPM2 information
+    how: shell
+    script: tpm2_getrandom 16 | hexdump
```

Comment:
Should this go in the final version? I thought it was just for testing that the TPM was correctly provisioned during development of the PR.

Reply:
I don't mind dropping it.